bundle/filefs.go

```diff
@@ -1,3 +1,4 @@
+//go:build go1.16
```

Comment:
https://stackoverflow.com/questions/68360688/whats-the-difference-between-gobuild-and-build-directives